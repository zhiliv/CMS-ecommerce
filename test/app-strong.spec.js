import { shallowMount } from '@vue/test-utils'
import H from '@/components/app/strong/strong.vue'

describe('app-strong.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(H, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent'
      },
      slots:{
        default: 'Содержимое'
      }
    })
  })
  it('Содержимое html для app-strong ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
