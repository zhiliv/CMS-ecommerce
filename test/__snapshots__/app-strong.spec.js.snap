// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-strong.js Содержимое html для app-strong  1`] = `
<strong
  class="simple simple-xl"
  id="indent"
  style="background: white"
>
  Содержимое
</strong>
`;
